src/WebWorker.ts

```diff
@@ -30,12 +30,33 @@ type ExceptionThrownCallback = (
 ) => void;
 type JSHandleFactory = (obj: Protocol.Runtime.RemoteObject) => JSHandle;
 
+/**
+ * The WebWorker class represents a [WebWorker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API).
+ * The events `workercreated` and `workerdestroyed` are emitted on the page object to signal the worker lifecycle.
```

Comment:
before this line you probably want to use a `@remarks` to split the summary from the examples, etc: https://api-extractor.com/pages/tsdoc/tag_remarks/